### PR TITLE
Move back to locked releases.

### PR DESCRIPTION
**Pull request by scirner22**
Moves back to locked releases for p8e and object-store. It would seem this is safer than using `latest` incase we release some changes that require something like added env vars. If we have the version locked here we will need to explicitly upgrade them and in the process we'd run it and see some env vars are missing.

### CODE REVIEW

File: docker-compose.yml [hunk 1]
```diff
@@ -67,7 +76,7 @@ services:
       - ./os_storage_bucket:/mnt/data
 
   p8e-migrate:
-    image: ghcr.io/provenance-io/p8e-migration:latest
+    image: ghcr.io/provenance-io/p8e-migration:0.8.2
```
Comment (rchaing-figure) [inline docker-compose.yml]: 0.8.3

File: docker-compose.yml [hunk 2]
```diff
@@ -81,7 +90,7 @@ services:
     command: migrate
 
   p8e:
-    image: ghcr.io/provenance-io/p8e-api:latest
+    image: ghcr.io/provenance-io/p8e-api:0.8.2
```
Comment (rchaing-figure) [inline docker-compose.yml]: 0.8.3

File: docker-compose.yml [hunk 3]
```diff
@@ -97,7 +106,7 @@ services:
       - 5002:8080
 
   p8e-webservice:
-    image: ghcr.io/provenance-io/p8e-api-webservice:latest
+    image: ghcr.io/provenance-io/p8e-api-webservice:0.8.2
```
Comment (rchaing-figure) [inline docker-compose.yml]: 0.8.3

**Review by piercetrey-figure — APPROVED** [review]
looks good once the 0.8.3 change is made

File: docker-compose.yml (hunk 1 above)
Comment (scirner22) [reply docker-compose.yml]: 👍

**Pull request merged**